custom_components/tesla_custom/climate.py

```diff
@@ -20,6 +20,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 SUPPORT_HVAC = [HVAC_MODE_HEAT_COOL, HVAC_MODE_OFF]
+import pprint
```

Comment:
nit: I think you're adding an import but not really using it

Reply:
fixed